README.md

```diff
@@ -236,12 +236,12 @@ _At this time, dynamic theming and localization is only supported for the `telem
 - Update the theme environment variables to point to your new folders:
     ```bash
     ASSETS_PATH='/src/assets/myTheme'
-    THEME_PATH='/src/assets/myTheme/theme'
+    THEME_PATH='/src/assets/theme/myTheme'
```

Comment:
The theme path was correct before

Reply:
I am actually going to simplify the theme folders in a follow up commit here. I don't like having multiple folders to hold the theme. Makes it harder to manage the PR's back into Ottehr